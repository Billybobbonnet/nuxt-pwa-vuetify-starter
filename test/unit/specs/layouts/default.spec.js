import { createLocalVue, shallowMount } from "@vue/test-utils";
import Vuetify from "vuetify";
import DefaultLayout from "~/layouts/default";

describe("Default layout", () => {
  let wrapper;

  beforeAll(() => {
    const localVue = createLocalVue();

    localVue.use(Vuetify);

    wrapper = shallowMount(DefaultLayout, {
      localVue,
      stubs: ["nuxt"]
    });
  });

  it("renders to a snapshot", () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
