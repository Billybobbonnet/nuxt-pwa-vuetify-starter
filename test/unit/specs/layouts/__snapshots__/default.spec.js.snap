// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default layout renders to a snapshot 1`] = `
<vapp-stub
  id="app"
>
  <vnavigationdrawer-stub
    app="app"
    fixed="fixed"
    height="100%"
    minivariantwidth="80"
    mobilebreakpoint="1264"
    value="true"
    width="300"
  >
    <vlist-stub>
      <vlisttile-stub
        activeclass="primary--text"
        exact="exact"
        router="router"
        to="/"
      >
        <vlisttileaction-stub>
          <vicon-stub />
        </vlisttileaction-stub>
         
        <vlisttilecontent-stub>
          <vlisttiletitle-stub />
        </vlisttilecontent-stub>
      </vlisttile-stub>
      <vlisttile-stub
        activeclass="primary--text"
        exact="exact"
        router="router"
        to="/inspire"
      >
        <vlisttileaction-stub>
          <vicon-stub />
        </vlisttileaction-stub>
         
        <vlisttilecontent-stub>
          <vlisttiletitle-stub />
        </vlisttilecontent-stub>
      </vlisttile-stub>
    </vlist-stub>
  </vnavigationdrawer-stub>
   
  <vtoolbar-stub
    app="app"
    color="primary"
    dark="dark"
    fixed="fixed"
    scrollthreshold="300"
  >
    <vtoolbarsideicon-stub />
     
    <vbtn-stub
      activeclass="v-btn--active"
      icon="true"
      tag="button"
      type="button"
    >
      <vicon-stub />
    </vbtn-stub>
     
    <vbtn-stub
      activeclass="v-btn--active"
      icon="true"
      tag="button"
      type="button"
    >
      <vicon-stub />
    </vbtn-stub>
     
    <vbtn-stub
      activeclass="v-btn--active"
      icon="true"
      tag="button"
      type="button"
    >
      <vicon-stub />
    </vbtn-stub>
     
    <vtoolbartitle-stub />
     
    <vbtn-stub
      activeclass="v-btn--active"
      icon="true"
      tag="button"
      type="button"
    />
  </vtoolbar-stub>
   
  <vcontent-stub
    tag="main"
  >
    <vcontainer-stub
      tag="div"
    >
      <nuxt-stub />
    </vcontainer-stub>
  </vcontent-stub>
   
  <vnavigationdrawer-stub
    fixed="fixed"
    height="100%"
    minivariantwidth="80"
    mobilebreakpoint="1264"
    right="true"
    temporary="true"
    width="300"
  >
    <vlist-stub>
      <vlisttile-stub
        activeclass="primary--text"
      >
        <vlisttileaction-stub>
          <vicon-stub
            light="light"
          />
        </vlisttileaction-stub>
         
        <vlisttiletitle-stub />
      </vlisttile-stub>
    </vlist-stub>
  </vnavigationdrawer-stub>
</vapp-stub>
`;
